packages/scripts/src/commands/app-data/convert/processors/flowParser/parsers/default.parser.spec.ts

```diff
@@ -1,3 +1,135 @@
-describe("default Parser", () => {
-  it("TODO - add tests", () => expect(true).toEqual(true));
+import { FlowTypes } from "data-models";
+import { DefaultParser } from "./default.parser";
+
+const testInputSources = {
+  data_list: { test_data_list: [{ id: 1 }, { id: 2 }, { id: 3 }] },
+};
+
+describe("Default Parser", () => {
+  it("Cleans field values - removes whitespace within empty strings", () => {
```

Comment:
I think this description could be more accurate, something like: "Cleans field values - handles strings consisting only of whitespace"